src/bin/data/mod.rs

```diff
@@ -0,0 +1,148 @@
+use std::collections::{HashMap, HashSet};
+use serde::{Serialize, Deserialize};
+use std::hash::Hash;
+
+#[derive(Serialize, Deserialize)]
+pub struct Database {
+    /*
```

Comment:
using 2 asteriscs instead of 1 will result in a doc comment

```rs
/*
 * Non doc comment
 */
 
/**
 * Doc comment
 */
```

Reply:
Sorry, that comment is meant to explain how the internal IDs should be used and the logic behind them, and isn't meant to explain the database itself

Reply:
oh okay!
Just thought you missed it, dw